manifests/provider/systemd.pp

```diff
@@ -0,0 +1,42 @@
+#
```

Comment:
puppet-strings will render the summary of private classes in the class lists, so we can add a description

```suggestion
# @summary Manage the systemd components of a wireguard setup
```